models/cow_protocol/arbitrum/cow_protocol_arbitrum_batches.sql

```diff
@@ -0,0 +1,125 @@
+{{  config(
+        alias = 'arbitrum_batches',
+
```

Comment:
We need the `schema` here, and probably we can put `arbitrum` inside the `schema`:
```suggestion
        schema = 'cow_protocol_arbitrum',
        alias = 'batches',
```